oak-run-commons/src/main/java/org/apache/jackrabbit/oak/run/cli/Options.java

```diff
@@ -80,6 +80,15 @@ public OptionSet parseAndConfigure(OptionParser parser, String[] args, boolean c
         if (checkNonOptions) {
             checkNonOptions();
         }
+        CommonOptions commonOpts = getCommonOpts();
+        if (!commonOpts.isHelpRequested() && !commonOpts.isContinueMissingDep()) {
```

Comment:
I think it's strange (unexpected) to do the check in parseAndConfigure. I wouldn't expect this method to throw an exception.

Reply:
moved